index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Eastern Designs</title>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="style.css" rel="stylesheet"/>
+  <link href="navbar.css" rel="stylesheet"/>
+  <script src="https://kit.fontawesome.com/a012cea4ea.js" crossorigin="anonymous"></script>

```

Comment:
- [ ] Kindly move the script to the last line before the closing `</ body>` tag. This is a good semantic arrangement.